Controller/ConnectController.php

```diff
@@ -282,7 +308,15 @@ private function authenticateUser(Request $request, UserInterface $user, string
         $token->setUser($user);
         $token->setAuthenticated(true);
 
-        $this->get('security.token_storage')->setToken($token);
+        /** @var TokenStorageInterface $tokenStorage */
+        $tokenStorage = $this->get('security.token_storage');
+
+        // Handle Session deprecations in Symfony 5.3+
+        if (Kernel::VERSION_ID >= 50300 && method_exists($tokenStorage, 'disableUsageTracking')) {
+            $tokenStorage->disableUsageTracking();
```

Comment:
can you explain why exactly we need to call this to prevent/handle a deprecation?

Reply:
Reverted, it was taken from so external repository doing similar update, for not it's not needed.